src/sentry/static/sentry/app/views/settings/team/teamProjects.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import createReactClass from 'create-react-class';
+import {Box} from 'grid-emotion';
+import Reflux from 'reflux';
+
+import ApiMixin from '../../../mixins/apiMixin';
+import ProjectsStore from '../../../stores/projectsStore';
+import TeamStore from '../../../stores/teamStore';
+import IndicatorStore from '../../../stores/indicatorStore';
+import TeamActions from '../../../actions/teamActions';
+
+import Button from '../../../components/buttons/button';
+import LoadingError from '../../../components/loadingError';
+import OrganizationState from '../../../mixins/organizationState';
+import ProjectListItem from '../components/settingsProjectItem';
+import Panel from '../components/panel';
+import PanelItem from '../components/panelItem';
+import PanelHeader from '../components/panelHeader';
+
+import {sortProjects} from '../../../utils.jsx';
+import {t} from '../../../locale';
+
+const TeamProjects = createReactClass({
+  displayName: 'TeamProjects',
+  mixins: [
+    ApiMixin,
+    OrganizationState,
+    Reflux.listenTo(ProjectsStore, 'onProjectUpdate'),
+    Reflux.listenTo(TeamStore, 'onTeamUpdate'),
+  ],
+
+  getInitialState() {
+    let team = TeamStore.getBySlug(this.props.params.teamId);
+    return {
+      allProjects: ProjectsStore.getAll(),
+      error: false,
+      projectListLinked: team ? team.projects : [],
+    };
+  },
+
+  componentWillReceiveProps(nextProps) {
+    let params = this.props.params;
+    if (
+      nextProps.params.teamId !== params.teamId ||
+      nextProps.params.orgId !== params.orgId
+    ) {
+      this.setState(this.getInitialState());
+    }
+  },
+
+  onProjectUpdate() {
+    this.setState({
+      allProjects: ProjectsStore.getAll(),
+    });
+  },
+
+  onTeamUpdate() {
+    this.setState({
+      projectListLinked: TeamStore.getBySlug(this.props.params.teamId).projects,
+    });
+  },
+
+  linkProject(project, value) {
+    let {orgId, teamId} = this.props.params;
+    this.api.request(`/projects/${orgId}/${project.slug}/teams/${teamId}/`, {
+      method: value === 'Add' ? 'POST' : 'DELETE',
+      success: () => {
+        let team = TeamStore.getBySlug(this.props.params.teamId);
+        if (value == 'Add') {
+          team.projects = [...team.projects, project];
+        } else {
+          team.projects = team.projects.filter(({id}) => id != project.id);
+        }
+        TeamActions.updateSuccess(0, teamId, team);
+      },
+      error: e => {
+        IndicatorStore.addError("Wasn't able to change project association.");
+      },
+    });
+  },
+
+  projectPanelcontents(projects, direction) {
+    return sortProjects(projects).map((project, i) => (
+      <PanelItem key={i} align="center">
+        <Box w={1 / 2} p={2}>
+          <ProjectListItem project={project} organization={this.context.organization} />
+        </Box>
+        <Box w={1 / 2} p={2} style={{textAlign: 'right'}}>
```

Comment:
You can use `css` instead of `style`. 

I think you could also remove this, and instead make the previous `Box` sibling have `flex="1"`, which seems cleaner